recipes/rapidfuzz/all/conanfile.py

```diff
@@ -1,28 +1,63 @@
-from conans import ConanFile, tools
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import get, copy
+from conan.tools.layout import basic_layout
+from conan.tools.scm import Version
+import os
 
-required_conan_version = ">=1.33.0"
 
+required_conan_version = ">=1.52.0"
 
-class RapidFuzzConan(ConanFile):
+
+class PackageConan(ConanFile):
     name = "rapidfuzz"
-    description = "Rapid fuzzy string matching in C++ using the Levenshtein Distance "
-    topics = ("conan", "cpp", "levenshtein", "string-matching", "string-similarity", "string-comparison")
+    description = "Rapid fuzzy string matching in C++ using the Levenshtein Distance"
+    license = "MIT"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/maxbachmann/rapidfuzz-cpp"
-    license = "MIT"
+    topics = ("levenshtein", "string-matching", "string-similarity", "string-comparison", "header-only")
+    settings = "os", "arch", "compiler", "build_type"
     no_copy_source = True
 
     @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    def _min_cppstd(self):
+        return 17
```

Comment:
Should this be conditional on the version? it was not required for the old one 🤔